.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+## Description
+
+<!-- Short description of the change. Aim to reference the issue number of the GitHub issue. -->
+
+## Todo
+
+- [ ] PR has a meaningful title?
```

Comment:
Curious of the team's thoughts on starting to establish some guidelines for PR templates. Open to all discussion of this.

Reply:
Yep I am all for having some standards around PRs. This is good start.